gen3/bin/approve_vpcpeering_request.sh

```diff
@@ -1,23 +1,45 @@
 #!/bin/bash
-if [ $# -ne 1 ]; then
+if [ $# -lt 1 ]; then
     echo "USAGE: $0 name_of_child_vpc"
     exit 1
 fi
 vpcpeering_pending_acceptence=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --output text)
 if [ -n "$vpcpeering_pending_acceptence" ]; then
-child_vpc_name=$1
-vpcpeerconnid=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --query 'VpcPeeringConnections[*].VpcPeeringConnectionId' --output text)
-vpccidrblock=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --query 'VpcPeeringConnections[*].RequesterVpcInfo.CidrBlock' --output text)
 
-aws ec2 accept-vpc-peering-connection --vpc-peering-connection-id $vpcpeerconnid
-aws ec2 create-tags --resources $vpcpeerconnid --tags Key=Name,Value="VPC peering between $child_vpc_name and csoc_main_vpc"
-echo "The vpc peering connection request for id $vpcpeerconnid was accepted"
+    child_vpc_name=$1
+    vpcpeerconnid=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --query 'VpcPeeringConnections[*].VpcPeeringConnectionId' --output text)
+    vpccidrblock=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --query 'VpcPeeringConnections[*].RequesterVpcInfo.CidrBlock' --output text)
+    acceptervpcid=$(aws ec2 describe-vpc-peering-connections --filters Name=status-code,Values=pending-acceptance --query 'VpcPeeringConnections[*].AccepterVpcInfo.VpcId' --output text)
+    ROUTE_TABLES=(rtb-23b6685f rtb-rtb-7ee06301)
```

Comment:
Is this a hard coded route table ID? I guess that's fine since it looks like we've been hardcoding this all along but `rtb-rtb-7ee06301` should probably be `rtb-7ee06301` right?

Reply:
Yeah the route ID was hardcoded in the previous script so I left it just in case. I am not sure about route table id `rtb-rtb-7ee06301` I guess it is supposed to be `rtb-7ee06301`. It needs to be checked in CTDS CSOC.

Reply:
I think if you preserve what was hardcoded in the script before (i.e. https://github.com/uc-cdis/cloud-automation/pull/1534/files#diff-49fb1a0d6091e1ba2697cbea00f1fc6f8a04004d761459bf0184fe1ed1ef30ecL18)  that would be fine going forward as well. With that said I don't think `rtb-rtb-####` is a valid Route Table ID so it would make sense to correct that

Reply:
I considered the way to preserve hardcoded values while coding and tested it so it can work with the hardcoded value when the flag is not passed. About the route table ID, can you help me with getting value of the route table id ?

Reply:
I completely agree. I made sure while changing the code in the file to preserve the hardcoded value. If in case the flag is not passed, the script will use the hardcoded values. I also made change for route table ID as before.